catkin_ws/src/planner/src/testMission.py

```diff
@@ -2,11 +2,36 @@
 
 import rospy
 from substates.utility.controller import Controller
-from std_msgs.msg import Bool, Empty
+from substates.utility.state import StateTracker
+import unittest
+import rostest
 
-if __name__ == "__main__":
-    rospy.init_node("moveForward",log_level=rospy.DEBUG)
-    controls = Controller(rospy.Time(0))
-    controls.move((0,0,0))
+ 
+## A sample python unit test
+class Test1(unittest.TestCase):
+    ## test 1 == 1
+    def test__simple_movement(self): # only functions with 'test_'-prefix will be run!
```

Comment:
while we're at it lets test some more complex orientations/position changes (can be done in a further PR, maybe lets make an issue out of it if we want to give this to a new member)

Reply:
lets save this for a new member